streams/config.go

```diff
@@ -2,23 +2,19 @@ package streams
 
 import (
 	"errors"
+	"github.com/jpillora/backoff"
 	"time"
 )
 
 type StreamsConfig struct {
-	Region               string        `config:"region"`
-	DeliveryStreamName   string        `config:"stream_name"`
-	PartitionKey         string        `config:"partition_key"`
-	PartitionKeyProvider string        `config:"partition_key_provider"`
-	BatchSize            int           `config:"batch_size"`
-	MaxRetries           int           `config:"max_retries"`
-	Timeout              time.Duration `config:"timeout"`
-	Backoff              backoff       `config:"backoff"`
-}
-
-type backoff struct {
-	Init time.Duration
-	Max  time.Duration
+	Region               string          `config:"region"`
+	DeliveryStreamName   string          `config:"stream_name"`
+	PartitionKey         string          `config:"partition_key"`
+	PartitionKeyProvider string          `config:"partition_key_provider"`
+	BatchSize            int             `config:"batch_size"`
+	MaxRetries           int             `config:"max_retries"`
+	Timeout              time.Duration   `config:"timeout"`
+	Backoff              backoff.Backoff `config:"backoff"`
```

Comment:
This is taken from filebeat.yml?

Reply:
This is the struct that maps how `filebeat.yml` values translate to Golang values.